build_tools/circle/build_wheels_arm.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+set -x
+
+sudo add-apt-repository --remove ppa:ubuntu-toolchain-r/test
```

Comment:
Why is this PPA needed?

Reply:
euh.... this command remove a repository from the apt-repository... I used the azure ubuntu installation as model and it was used [there](https://github.com/scikit-learn/scikit-learn/blob/fac31e727947ad53f2ed107f58a10b56b165cee7/build_tools/azure/install.sh#L67) ... I have removed it.